test_scripts/API/RemoveUrlParameterMaxLength/001_OnSystemRequest.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0200-Removing-URL-Param-Max-Length.md
+--
+-- Description: Check processing of OnSystemRequest notification with different length of url
+--
+-- In case:
+-- 1. HMI sends OnSystemRequest notification with length of url is less than min value
+-- SDL does:
+-- - ignore notification and not send it to mobile app
+-- 2. HMI sends OnSystemRequest notification with in bound value in url parameter
+-- SDL does:
+-- - send notification with received url to mobile app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/RemoveUrlParameterMaxLength/commonRemoveUrlParameterMaxLength')
+
+--[[ Local Variables ]]
+local expected = 1
+local notExpected = 0
+local outOfMinLength = ""
+local minlength = "u"
```

Comment:
I think `minlength` implies an integer value. However, this value is used as a sample string for the test case
```suggestion
local minLengthString = "u"
```